lib/mayaUsd/ufe/UsdConnectionHandler.cpp

```diff
@@ -88,30 +88,96 @@ bool UsdConnectionHandler::createConnection(
     const Ufe::Attribute::Ptr& srcAttr,
     const Ufe::Attribute::Ptr& dstAttr) const
 {
-    PXR_NS::UsdAttribute srcUsdAttr = usdAttrFromUfeAttr(srcAttr);
-    PXR_NS::UsdAttribute dstUsdAttr = usdAttrFromUfeAttr(dstAttr);
+    UsdAttribute* srcUsdAttr = usdAttrFromUfeAttr(srcAttr);
+    UsdAttribute* dstUsdAttr = usdAttrFromUfeAttr(dstAttr);
 
-    if (!isConnected(srcUsdAttr, dstUsdAttr)) {
-        return dstUsdAttr.AddConnection(srcUsdAttr.GetPath());
+    if (!srcUsdAttr || !dstUsdAttr
```

Comment:
if `srcUsdAttr` or `dstUsdAttr` is null then the code should report an error e.g. `TF_RUNTIME_ERROR`